src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -549,18 +621,38 @@ private bool Parse(string[] args)
                     Reset = true;
                 }
 
+                string requireTimeSpanString = null;
+                if (argumentDictionary.TryGetValue("require", out requireTimeSpanString))
+                {
+                    TimeSpan requireTimeSpan;
+                    if (String.IsNullOrWhiteSpace(requireTimeSpanString))
+                    {
+                        RequireBridgeTimeSpan = DefaultRequireBridgeTimeSpan;
+                    }
+                    else if (TimeSpan.TryParse(requireTimeSpanString, out requireTimeSpan))
+                    {
+                        RequireBridgeTimeSpan = requireTimeSpan;
+                    }
+                    else
+                    {
+                        Console.WriteLine("The value \"{0}\" is not a valid TimeSpan", requireTimeSpanString);
+                        return false;
+                    }
+                }
+
                 return true;
             }
 
             private void ShowUsage()
             {
                 Console.WriteLine("Usage is: Bridge.exe [/ping] [/stop] [/stopIfLocal] [/allowRemote] [/remoteAddresses:x,y,z] [/{BridgeProperty}:value");
-                Console.WriteLine("   /ping            Pings the Bridge to check if it is running");
-                Console.WriteLine("   /stop            Stops the Bridge if it is running");
-                Console.WriteLine("   /stopIfLocal     Stops the Bridge if it is running locally");
-                Console.WriteLine("   /allowRemote     If starting the Bridge, allows access from other than localHost (default is localhost only)");
-                Console.WriteLine("   /reset           Releases all Brige resources without stopping Bridge");
-                Console.WriteLine("   /remoteAddresses If starting the Bridge, comma-separated list of addresses firewall rules will accept (default is 'LocalSubnet')");
+                Console.WriteLine("   /ping             Pings the Bridge to check if it is running");
+                Console.WriteLine("   /require:timespan Checks whether the Bridge is running and starts it if needed");
```

Comment:
NIT: it's not entire apparent from a cursory reading of Usage why we need to pass in a timespan